### ISSUE TITLE
[Bug]: Telegram private channel ID value exceeds MAX integer value

### ISSUE THREAD

**Issue by nAoxide** [openbullet/OpenBullet2#915]
### Version of the software

0.2.5

### Operating system

Windows 10

### Browser / Native

Native

### What happened?

I'm trying to add my private Telegram channel for hit output, but once I paste my channel ID, it automatically gets changed to the minimum Integer value "-2147483648".
I think data type need to be changed.
Here's an example for Telegram private channel ID "-1001927406021"

**Comment by openbullet** [comment]
Yeah I think this is because it should be a string instead of an integer. I'll fix it.

**Comment by openbullet** [comment]
Solved on current staging, will be part of next release
